applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian_UP.cpp

```diff
@@ -504,23 +505,6 @@ double& UpdatedLagrangianUP::CalculatePUCoefficient(double& rCoefficient, Genera
 }
 
 
-//************************************************************************************
-//************************************************************************************
-
-double& UpdatedLagrangianUP::CalculatePUDeltaCoefficient(double &rDeltaCoefficient, GeneralVariables & rVariables)
```

Comment:
Hi! Yes, it's true. Although, in all the cases performed with this type of stabilization this parameter has been always =1.  I think that this stabilization is not very "robust" for doing an study in the future changing this parameter, and for that I deleted it... But I don't know, may be it's better leave it as it was, just in case

Reply:
Ok, up to you then. If you think it will stay 1 then it will be better to clean up the code. Otherwise, I would maybe leave it for now.